src/main/java/io/quarkus/gizmo2/impl/ThisExpr.java

```diff
@@ -3,9 +3,8 @@
 import java.lang.constant.ClassDesc;
 
 import io.github.dmlloyd.classfile.CodeBuilder;
-import io.quarkus.gizmo2.Var;
 
-public final class ThisExpr extends Item implements Var {
+public final class ThisExpr extends Item {
```

Comment:
I still don't like this. With this change, `this` is the only expression that can always be reused many times without having some other kind of involved subtype to indicate that fact. There should be some kind of interface for it. Maybe even just a `This extends Expr`.

Reply:
That actually makes a lot of sense! I'll fix that.

Reply:
Done.